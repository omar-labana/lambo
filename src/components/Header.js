import React from 'react'

const Header = () => {
    return (
        <header className="flex items-center justify-between py-5">
            <div className="dancing-font text-6xl">Lambo</div>
            <ul className="flex gap-5 text-white text-2xl">
                <li className="border-b-4 pb-1 px-1 border-yellow">Home</li>
                <li className="text-gray-600">Today's Dish</li>
            </ul>
        </header>
    )
}

export default Header
