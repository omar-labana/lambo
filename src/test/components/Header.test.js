import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Header from '../../components/Header';

describe('Header Component', () => {
  const wrapper = shallow(<Header />);

  it('exists', () => {
    expect(wrapper.find('Header')).toBeTruthy();
  });

  it('has to match snap', () => {
    const home = renderer.create(<Header />);
    expect(home).toMatchSnapshot();
  });
});
