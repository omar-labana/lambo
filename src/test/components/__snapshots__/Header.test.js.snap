// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component has to match snap 1`] = `
<header
  className="flex items-center justify-between py-5"
>
  <div
    className="dancing-font text-6xl"
  >
    Lambo
  </div>
  <ul
    className="flex gap-5 text-white text-2xl"
  >
    <li
      className="border-b-4 pb-1  border-yellow"
    >
      Home
    </li>
    <li
      className="text-gray-600"
    >
      Today's Dish
    </li>
  </ul>
</header>
`;
